src/pages/token-metadata/mint.md

```diff
@@ -6,17 +6,17 @@ description: Learn how to mint NFTs, SFTs and Programmable NFTs (a.k.a. Assets)
 
 As we discussed in the [Token Metadata overview](/token-metadata), digital assets on Solana are composed of several on-chain accounts and off-chain data describing the token. On this page, we'll go over the process of minting these assets. {% .lead %}
 
-## The minting process
+## The minting process [#](#the-minting-process)
 
-Whether we want to mint a Fungible, Semi-Fungible or Non-Fungible asset, the overall process is the same:
+Whether we want to mint a Fungible, Semi-Fungible or Non-Fungible NFT, the overall process is the same:
 
-1. **Upload off-chain data.** First, we must ensure our off-chain data is ready. This means we must have a JSON file stored somewhere that describes our asset. It doesn't matter how or where that JSON file is stored, as long as it's accessible via a **URI**.
-2. **Create on-chain accounts.** Then, we must create the on-chain accounts that will hold our asset's data. Which exact accounts will be created depends on the **Token Standard** of our asset, but in all cases, a **Metadata** account will be created and will store the **URI** of our off-chain data.
+1. **Upload off-chain data.** First, we must ensure our off-chain data is ready. This means we must have a JSON file stored somewhere that describes our NFT. It doesn't matter how or where that JSON file is stored, as long as it's accessible via a **URI**.
+2. **Create on-chain accounts.** Then, we must create the on-chain accounts that will hold our NFT's data. Which exact accounts will be created depends on the **Token Standard** of our NFT, but in all cases, a **Metadata** account will be created and will store the **URI** of our off-chain data.
```

Comment:
Here's an example where the sentence becomes ambiguous about the token standard when changing asset to NFT.